app/assets/stylesheets/conversation_show.scss

```diff
@@ -32,16 +32,46 @@ margin: 0 0 5px 0;
   text-align: center;
 }
 
-.chatboxmessagecontent {
-  padding-left: 5px;
-  padding-bottom: 10px;
+// .chatboxmessagecontent {
+//   padding-left: 5px;
+//   padding-bottom: 10px;
+// }
+
+.chatboxmessagecontent time {
+    font-size: 9px;
+    color: #ccc;
 }
+
 .self {
-  color: blue;
+  justify-content: flex-end;
+  align-items: flex-end;
 }
 
-.other {
-  color: red;
+.other time{
+  position: absolute;
+  clear: both;
+  border: 5px solid white;
+  border-left-color: transparent;
+  border-bottom-color: transparent;
+}
+
+.chatboxmessagecontent {
+    padding: 10px;
+    border-radius: 3px;
+    word-wrap: break-word;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.2);
+}
+
+.self .chatboxmessagecontent {
+  background-color: $jumbo;
+  order: 1;
```

Comment:
Properties should be ordered background-color, border-bottom-right-radius, border-radius, order